x/incentives/client/cli/query.go

```diff
@@ -317,6 +318,47 @@ $ %s query incentives upcoming-gauges
 	return cmd
 }
 
+// GetCmdActiveGaugesPerDenom returns active gauges for specified denom.
+func GetCmdUpcomingGaugesPerDenom() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upcoming-gauges-per-denom [denom]",
+		Short: "Query scheduled gauges per denom",
+		Long: strings.TrimSpace(
```

Comment:
nit: for concise/straight forward CLI handlers like this, I'd argue `Long` is not necessary as it does not offer any additional help or insight.

Reply:
bump @xBalbinus